packages/datadog-plugin-aerospike/src/index.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const { storage } = require('../../datadog-core')
+const DatabasePlugin = require('../../dd-trace/src/plugins/database')
+
+class AerospikePlugin extends DatabasePlugin {
+  static get id () { return 'aerospike' }
+  static get operation () { return 'command' }
+  static get system () { return 'aerospike' }
+
+  bindStart (ctx) {
+    const { commandName, commandArgs } = ctx
+    const resourceName = commandName.slice(0, commandName.indexOf('Command'))
+    const store = storage.getStore()
+    const childOf = store ? store.span : null
+    let ns, set, key, bin, index
+    let meta = {}
+
+    if (resourceName.includes('Index')) {
+      [ns, set, bin, index] = commandArgs
+      meta = getMetaForIndex(ns, set, bin, index)
+    } else if (resourceName === 'Query') {
+      const { ns: queryNs, set: querySet } = commandArgs[2]
+      meta = getMetaForQuery({ ns: queryNs, set: querySet })
+    } else if (isKeyObject(commandArgs[0])) {
+      ({ ns, set, key } = commandArgs[0])
+      meta = getMetaForKey(ns, set, key)
+    }
+
+    const span = this.startSpan(this.operationName(), {
+      childOf,
+      service: this.serviceName({ pluginConfig: this.config }),
+      type: 'aerospike',
+      kind: 'client',
+      resource: resourceName,
+      meta
+    }, false)
+
+    ctx.parentStore = store
+    ctx.currentStore = { ...store, span }
+
+    return ctx.currentStore
+  }
+
+  bindAsyncStart (obj) {
+    obj.currentStore.span.finish()
+    return obj.parentStore
+  }
+
+  end (ctx) {
+    if (ctx.result) {
+      ctx.currentStore.span.finish()
+    }
+  }
+
+  error (ctx) {
+    const error = ctx.error
+    const span = ctx.currentStore.span
+    span.setTag('error', error)
+  }
+}
+
+function getMetaForIndex (ns, set, bin, index) {
+  return {
+    'aerospike.namespace': ns,
+    'aerospike.setname': set,
```

Comment:
Are these tags specified anywhere? And if not, do they match existing implementations?

Reply:
got it from the dotnet snapshots, since there is no spec for aerospike i believe both dotnet and java do something a little different

Reply:
dotnet snapshots

https://github.com/datadog/dd-trace-dotnet/blob/6aab5e1b02bec9c9b68a33cd06cc9e7a774f14de/tracer/test/snapshots/AerospikeTests.SchemaV0.verified.txt#L1